### DISCUSSION TITLE
Operator Lifting

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#146] [Q&A]
Is there any way to lift operator signatures in operator signatures like as we could have for lambdas?

**Comment by h3rald** [comment]
Uhm... wrap it in a lambda maybe? 

Would that work?

**Reply by ghost** [reply]
Lambdas are not same as operators. Because in operators we can describe expectations but we can't do that in lambdas.

I tried adding an operator signature where is in operator signature to ROOT with using `parent-scope` but i failed.

**Reply by h3rald** [reply]
Did you try with `with`?

What are you trying to do with all this anyway? What do you want to achieve? Why would you want to define something on ROOT from another scope?

Just trying to understand your use case.
